src/bg_daemon/main.m

```diff
@@ -0,0 +1,50 @@
+#import <Foundation/Foundation.h>
+#import <AppKit/AppKit.h>
+
+int main(int argc, const char * argv[]) {
+  int ret = 0;
+
+  @autoreleasepool {
```

Comment:
I'm not familiar with this. I'm guessing this is some ARC statement. 

Is the code inside expected to be only one space of indentation deeper?

I'm not sure about this, so it's your call.